tests/llm/accuracy_conformance.py

```diff
@@ -43,75 +97,108 @@ def init_test_scope():
-        ov_model.save_pretrained(ov_model_path)
-        tokenizer.save_pretrained(ov_model_path)
-        del ov_model
+        ov_tokenizer, ov_detokenizer = convert_tokenizer(tokenizer, with_detokenizer=True)
+
+        int8_model_path = get_model_path(model_id, PREC_INT8)
+        # If directory exists, do not quantize again. It is for local development purpose. CI should re-generate it everytime.
+        if not os.path.exists(int8_model_path):
+            logger.info(f'Saving int8 OpenVINO model: {int8_model_path}')
+            ov_model = OVModelForCausalLM.from_pretrained(model_path, load_in_8bit=True)
+            ov_model.save_pretrained(int8_model_path)
+            tokenizer.save_pretrained(int8_model_path)
+            del ov_model
+            save_model(ov_tokenizer, os.path.join(int8_model_path, "openvino_tokenizer.xml"))
+            save_model(ov_detokenizer, os.path.join(int8_model_path, "openvino_detokenizer.xml"))
         gc.collect()
 
-        quantization_config = OVWeightQuantizationConfig(bits=4, ratio=0.5)
-        quantized_model = OVModelForCausalLM.from_pretrained(
-            model_path, quantization_config=quantization_config
-        )
-        quantized_model_path = os.path.join(tmp_dir, model_type + "_ov_int4")
-        quantized_model.save_pretrained(quantized_model_path)
-        tokenizer.save_pretrained(quantized_model_path)
-        del quantized_model
+        int4_model_path = get_model_path(model_id, PREC_INT4)
+        if not os.path.exists(int4_model_path):
+            logger.info(f'Quantizing model to INT4: {int4_model_path}')
+            quantization_config = OVWeightQuantizationConfig(bits=4, ratio=0.8)
+            quantized_model = OVModelForCausalLM.from_pretrained(
+                model_path, quantization_config=quantization_config
+            )
+            quantized_model.save_pretrained(int4_model_path)
+            tokenizer.save_pretrained(int4_model_path)
+            del quantized_model
+            save_model(ov_tokenizer, os.path.join(int4_model_path, "openvino_tokenizer.xml"))
+            save_model(ov_detokenizer, os.path.join(int4_model_path, "openvino_detokenizer.xml"))
         gc.collect()
 
         set_seed(42)
-        evaluator = wwb.Evaluator(
-            base_model=model, tokenizer=tokenizer, num_samples=NUMBER_OF_SAMPLES
-        )
-        gt_path = os.path.join(tmp_dir, model_type + "_gt.json")
-        evaluator.dump_gt(gt_path)
-        [
-            test_scope.append((ov_model_path, model_type, "INT8", gt_path, device))
-            for device in DEVICES
-        ]
-        [
-            test_scope.append((ov_model_path, model_type, "INT4", gt_path, device))
-            for device in DEVICES
-        ]
+        gt_path = get_gt_path(model_id)
+        if not os.path.exists(gt_path):
+            evaluator = wwb.Evaluator(
+                base_model=model, tokenizer=tokenizer, num_samples=NUMBER_OF_SAMPLES, use_chat_template=True
```

Comment:
it seems to me you also need to set chat_template depending on presence it in tokenizer to align results with evaluator from test function